tidb-server/server/util_test.go

```diff
@@ -15,9 +15,40 @@ package server
 
 import (
 	. "github.com/pingcap/check"
+	mysql "github.com/pingcap/tidb/mysqldef"
 )
 
 var _ = Suite(&testUtilSuite{})
 
 type testUtilSuite struct {
 }
+
+func (s *testUtilSuite) TestDumpBinaryTime(c *C) {
+	tests := []struct {
+		Input  string
+		Expect []byte
+	}{
+		{"0000-00-00 00:00:00.0000000", []byte{11, 1, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0}},
+		{"0000-00-00", []byte{4, 1, 0, 1, 1}},
+		{"0000-00-00 00:00:00.0000000", []byte{0}},
```

Comment:
It is a little confused to use tests variables, for different item will do different test check?
I think define separate variables it better.